<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CGridLimitsDialog</class>
 <widget class="QDialog" name="CGridLimitsDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>257</width>
    <height>245</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Grid limits</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../DyssolGUI/Resources.qrc">
    <normaloff>:/Icons/WinIcon</normaloff>:/Icons/WinIcon</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBoxX">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="labelXDescr">
        <property name="text">
         <string>X</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4">
        <item>
         <widget class="QLabel" name="labelXMin">
          <property name="text">
           <string>MIN</string>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditXMin"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,4">
        <item>
         <widget class="QLabel" name="labelXMax">
          <property name="text">
           <string>MAX</string>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditXMax"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="checkLogX">
        <property name="text">
         <string>Plot in log-scale</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxY">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="labelYDescr">
        <property name="text">
         <string>Y</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,4">
        <item>
         <widget class="QLabel" name="labelYMin">
          <property name="text">
           <string>MIN</string>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditYMin"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,4">
        <item>
         <widget class="QLabel" name="labelYMax">
          <property name="text">
           <string>MAX</string>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditYMax"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="checkLogY">
        <property name="text">
         <string>Plot in log-scale</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="buttonOk">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonApply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditXMin</tabstop>
  <tabstop>lineEditXMax</tabstop>
  <tabstop>lineEditYMin</tabstop>
  <tabstop>lineEditYMax</tabstop>
  <tabstop>buttonOk</tabstop>
  <tabstop>buttonCancel</tabstop>
  <tabstop>buttonApply</tabstop>
 </tabstops>
 <resources>
  <include location="../../DyssolGUI/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
